composer.json

```diff
@@ -16,7 +16,7 @@
     "php": "^7.3 || ^8.0",
     "ext-json": ">=1.3.7",
     "ezimuel/ringphp": "^1.1.2",
-    "psr/log": "^1|^2"
+    "psr/log": "^1|^3"
```

Comment:
this has nothing todo with the coding standard, please revisit your PR.

Reply:
`PSR`'s are no coding standards? Really?

Reply:
psr-2 and psr-12 yes, but `psr/log` is https://www.php-fig.org/psr/psr-3/ , a different standard

Reply:
:facepal